recipes/tt-metalium/meta.yaml

```
@@ -0,0 +1,67 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+# If your package is python based, we recommend using Grayskull to generate it instead:
+# https://github.com/conda-incubator/grayskull
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 16 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "tt-metalium" %}
+{% set version = "0.55.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/tenstorrent/tt-metal/releases/download/v{{ version }}-rc13/tt-metalium.tar.gz
+    sha256: 5b4b4454c6fe60e91730b676ccc126fdc41149738e06f3e8d34bb9979bf30259
```

Since this only uses one source, would suggest writing it like so

```suggestion
source:
  url: https://github.com/tenstorrent/tt-metal/releases/download/v{{ version }}-rc13/tt-metalium.tar.gz
  sha256: 5b4b4454c6fe60e91730b676ccc126fdc41149738e06f3e8d34bb9979bf30259
```